scripts/generate-AUTHORS.py

```diff
@@ -23,104 +23,8 @@
 	# Bots
```

Comment:
Not directly part of this patch but in L21 ```"unknown <d0t@.(none)>",``` is another git configuration screwup by me and should probably be aliased accordingly.